app/assets/javascripts/views/project_view.js

```diff
@@ -12,6 +15,9 @@ module.exports = Backbone.View.extend({
     this.model.stories.on('reset', this.addAll);
     this.model.stories.on('all', this.render);
     this.model.on('change:userVelocity', this.addAll);
+    this.listenTo(this.model, 'change:inverse_flow', this.prepareColumns);
+
+    this.prepareColumns();
```

Comment:
`prepareColumns` basically rewrite the `$el` based on the template. It doesn't render the columns contents, it just prepares the html element. Do you have any suggestions that could be less confusing?